docs/dev_install.rst

```diff
@@ -19,18 +19,22 @@ Installing Dependencies
 
   .. code-block:: bash
 
-    python3 -m pip install --user setuptools astpretty astor pybind11 Pillow dill
+    python3 -m pip install --user setuptools wheel astor pybind11 Pillow dill
     python3 -m pip install --user pytest pytest-rerunfailures pytest-xdist yapf
     python3 -m pip install --user numpy GitPython coverage colorama autograd
 
+
 - Make sure you have ``clang`` with version >= 7
 
-  * On Windows: Download ``clang-8`` via `this link <https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe>`_.
+  * On Windows: Download from `clang-8 <https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe>`_ or 
+    `clang-10 <https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/clang-10.0.0-win.zip>`_.
     Make sure you add the ``bin`` folder containing ``clang.exe`` to the ``PATH`` environment variable.
 
   * On OS X: you don't need to do anything.
 
-  * On Ubuntu, execute ``sudo apt install libtinfo-dev clang-8``.
+  * On Ubuntu, execute ``sudo apt install clang-8``.
```

Comment:
I thought `libtinfo-dev` is still needed on Ubuntu?

Reply:
We may move it to another section, this section is for clang, also other Linux dist do need this stupid lib too. In fact, I found my dev install don't need this at all... @yuanming-hu Is our Linux buildbot running on Ubuntu-16.04 that caused this issue? `ldd` shows that my dev build use `libncursesw.so.6` instead of `libtinfo.so.5`.
```
130 root@archlinux ~/taichi (git)-[ti-init-toggle-advanced-optimization] # ldd build/libtaichi_core.so                                                                                     :(
	linux-vdso.so.1 (0x00007ffec11ec000)
	libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007fea2b633000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007fea2b456000)
	libX11.so.6 => /usr/lib/libX11.so.6 (0x00007fea2b315000)
	librt.so.1 => /usr/lib/librt.so.1 (0x00007fea2b30a000)
	libm.so.6 => /usr/lib/libm.so.6 (0x00007fea2b1c5000)
	libz.so.1 => /usr/lib/libz.so.1 (0x00007fea2b1ab000)
	libdl.so.2 => /usr/lib/libdl.so.2 (0x00007fea2b1a3000)
	libncursesw.so.6 => /usr/lib/libncursesw.so.6 (0x00007fea2b132000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007fea2af6b000)
	/usr/lib64/ld-linux-x86-64.so.2 (0x00007fea2efae000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007fea2af51000)
	libxcb.so.1 => /usr/lib/libxcb.so.1 (0x00007fea2af27000)
	libXau.so.6 => /usr/lib/libXau.so.6 (0x00007fea2af22000)
	libXdmcp.so.6 => /usr/lib/libXdmcp.so.6 (0x00007fea2af18000)
root@archlinux ~/taichi (git)-[ti-init-toggle-advanced-optimization] # ldd /usr/lib/python3.8/site-packages/taichi/lib/libtaichi_core.so  
	linux-vdso.so.1 (0x00007ffea8717000)
	libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007fde51926000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007fde51749000)
	libX11.so.6 => /usr/lib/libX11.so.6 (0x00007fde51608000)
	librt.so.1 => /usr/lib/librt.so.1 (0x00007fde515fd000)
	libm.so.6 => /usr/lib/libm.so.6 (0x00007fde514b8000)
	libz.so.1 => /usr/lib/libz.so.1 (0x00007fde5149e000)
	libdl.so.2 => /usr/lib/libdl.so.2 (0x00007fde51496000)
	libtinfo.so.5 => /usr/lib/libtinfo.so.5 (0x00007fde51432000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007fde5126b000)
	/usr/lib64/ld-linux-x86-64.so.2 (0x00007fde55313000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007fde51251000)
	libxcb.so.1 => /usr/lib/libxcb.so.1 (0x00007fde51227000)
	libXau.so.6 => /usr/lib/libXau.so.6 (0x00007fde51222000)
	libXdmcp.so.6 => /usr/lib/libXdmcp.so.6 (0x00007fde51218000)
```